config/locales/fra/action.yml

```diff
@@ -1248,6 +1248,7 @@ fra:
     disease_percentage_evolution: "Evolution des maladies"
     display: "Afficher"
     display_help: "Afficher l’aide" #?
+    display_n_1: "Afficher N-1"
```

Comment:
Do we really want "N-1" and not something like "période précédente" ?

Reply:
The idea is to display data for the same selected period one year before, which may not be the previous financial year, not to mention it can also be a custom interval where "previous period" would become inaccurate